Makefile

```diff
@@ -44,13 +44,13 @@ release:
 
 start:
 	cp -f .env ./_build/prod/rel/blockchain_http/
-	./_build/prod/rel/blockchain_http/bin/blockchain_http foreground
+	./_build/prod/rel/blockchain_http/bin/blockchain_http daemon
```

Comment:
this is not a correct change.. the http workers need to run in foreground mode since this reflects how they get run in docker.